### ISSUE TITLE
[Personaje] Wiinter Corrupcion bugueada

### ISSUE THREAD

**Issue by Wiinter** [RoDNet/Bugtracker-Infinium-4.0.6a#1279]
Como mucho sabreis mi personaje principal es una bruja aflicción que siempre está dando la tabarra por aqui por el bugtracker. Pues estos últimos días mientras hacía mazmorras con los amigotes de la guild, me he dado cuenta de que el talento de aflicción, aflicción eterna, no funciona si estoy con ellos en grupo. En consecuencia no se renueva corrupción y tienes qe estar atento a todos los anteriores dots más ese y si alguien sabe algo del brujo aflicción sabrá que Corrupción es de los hechizos que más daño provoca por el hecho de estar siempre activa y en constante renuevo.

Puse un ticket el otro dia y un Maestro del Juego llamado Kran estuvo hablando conmigo y comprovando que era cierto y que solo pasa cuando estoy en grupo o banda, en cambio si estoy solo no. Entonces estuvo haciendo unas comprovaciones y me mandó a hablar con otro Maestro del Juego llamado Helkky, espero que haya escrito bien el nombre. Este último me dijo que los bugs hay qe postearlos en el bugtracker y aquí estoy con mi historia extraña.

Tengo que decir que mi viejo amigo brujo Evilknight, estuvo ayudando a Kran y a mi, para ver si a él le sucedía lo mismo y esto solo era problema mío. Tengo que decir que me he cambiado los talentos varias veces pero contínua igual. Me los cambio y van pero a la que desconecto o hago cualquier cosa vuelve el problema y lo más misterioso de todo es que solo me ocurre cuando estoy en grupo y con más gente a mi alrededor.

Espero que me puedan ayudar, porque es un problema bastante grave para un aflicción como yo.

Gracias de antemano y espero que no hay sido muy pesado leer todo ese parrafote.

Cualquier cosa podeis responder aqui o podeis encontrar ingame con mi pj Wiinter

**Comment by paellaencol** [comment]
Hay un pj brujo "Nicerdicer" de nuestra hermandad, al que le ocurre tres cuartas partes de lo mismo, en ocasiones la corrupción no se le renueva, y en otras sí. Además, él creo que tenía un problema con la armadura que en ocasiones se le desvanecía también.

**Comment by Wiinter** [comment]
Sí, lo de la armadura también me pasa, pero es un problema menor, porque con el power auras cuando no la tengo puesta me sale un dibujito en medio de la pantalla y me lo avisa.
Dile a tu compañero que se pase y comente por aqui :)

**Comment by nicerdicer** [comment]
Buenas soy el compañero de Paella en RoD, Nicerdicer Brujo afliccion Horda lvl 85. El problema de la armadura vil, se da en el momento en el que tras usar [Trueque de Alma] habilidad con la que se transfieren las aflicciones de un objetivo a otro tras usar [Trueque de alma: Exhalación], es en el momento en el que se usa esta ultima cuando el buff de la armadura desaparece, obligando a acabar el pull sin ella (en el caso de estar pendiente de concluir una rotación) o molestarse en renovar el buff... 
Concluyo apoyando tambien el reporte del bug en el talento de Aflicción Eterna, he estado haciendo averiguaciones, y me he fijado que a veces (y digo a veces por que no estoy muy seguro) lanzar [Poseer] sobre el afectado por la corrupción si que la renueva, si a esta aun le quedan mas de 10-15 segundos de duración, ya que han habido ocasiones en las que he terminado pulls en bosses largos sin perderla, pero siempre preocupandome de que corrupcion no bajara de 10 o menos segundos de duración restante. 
Aun asi, esto ultimo es una hipótesis, dado que aun teniendo el Addon Quartz se me escapa algun dot sin darme cuenta... 
Saludos y Suerte!!!

**Comment by Wiinter** [comment]
Mi problema ha surgido estos ultimos días, despues de la gran caida. Si estoy en grupo no se renueva corrupcion, lance lo que lance.
